.github/workflows/build-desktop-apps.yml

```diff
@@ -59,6 +58,7 @@ jobs:
           yarn message-system-sign-config
           yarn workspace @trezor/suite-data build:lib
           yarn workspace @trezor/connect-iframe build:lib
```

Comment:
this line is no needed very likely

Reply:
Well, the build was failing without the iframe, so it has to be there for the build to work for now, but I agree that it shouldn't be needed for the desktop app. Btw it's also required for the suite-web as the build was failing for that, too.